src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/SystemNavigationBar/SystemNavigationBarBehavior.shared.cs

```diff
@@ -0,0 +1,87 @@
+
+using System.Runtime.CompilerServices;
+using System.Runtime.Versioning;
+using CommunityToolkit.Maui.Core;
+using CommunityToolkit.Maui.Core.Platform;
+using CommunityToolkit.Maui.Extensions;
+
+namespace CommunityToolkit.Maui.Behaviors;
+
+/// <summary>
+/// <see cref="PlatformBehavior{TView,TPlatformView}"/> that controls the System navigation bar color
+/// </summary>
+[UnsupportedOSPlatform("Windows"), UnsupportedOSPlatform("MacCatalyst"), UnsupportedOSPlatform("MacOS"), UnsupportedOSPlatform("Tizen"), UnsupportedOSPlatform("iOS")]
+public class SystemNavigationBarBehavior : PlatformBehavior<Page>
```

Comment:
Since this is a platform-specific just for Android, I don't think this is the best way to do it. You should use the `OnPlatform` [API](https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.onplatform-1?view=net-maui-8.0), we don't have it implemented here but you can adapt the implementation we did for XCT.

[Here's a sample on how to use that API](https://learn.microsoft.com/en-us/dotnet/maui/ios/platform-specifics/page-safe-area-layout?view=net-maui-8.0)